test/unittests/libethcore/PrecompiledTest.cpp

```diff
@@ -656,6 +656,12 @@ namespace
 {
 void benchmarkPrecompiled(char const name[], vector_ref<const PrecompiledTest> tests, int n)
 {
+	if (!Options::get().all)
```

Comment:
There are also 2 SHA256 benchmarks. Search for `label("bench")` or `label("perf")`. Change the label to "bench" if "perf" is used.

Reply:
those are already work under --all option. anyway pushed the fix